CHANGELOG.rst

```diff
@@ -16,12 +16,74 @@ output of ``git log``
 
    0[1011] machine:dartGIT % git log > full_git_log.txt
```

Comment:
```suggestion
   % git log > full_git_log.txt
```
the complex prompt stuff is slightly confusing.

Reply:
Says you :)!  Having just a percent to indicate that this is a unix prompt is a bit spartan for my taste. People might just copy the thing in its entirety.